packages/format-library/src/link/inline.js

```diff
@@ -221,37 +250,36 @@ class InlineLinkUI extends Component {
 		const showInput = isShowingInput( this.props, this.state );
 
 		return (
-			<PositionedAtSelection
```

Comment:
Minor thing, but since `PositionedAtSelection` is exposed and will be introduced this WordPress release, could we removed it?

We have one more similar use of it:

https://github.com/WordPress/gutenberg/blob/7964f381e07e06b2ac13698fd1465a4d0c16da1b/packages/format-library/src/image/index.js#L116-L120

So I would either adjust that one too, or temporarily mark the component as unstable.

Reply:
(If you want I can also take this over.)

Reply:
Hm, I thought we'd already been exposing the component, so I didn't think too much about it. I might agree that if we're uncertain about its future, to mark is as unstable.

Reply:
I think it doesn't have much purpose if `Popover` has this ability as well.